src/components/shared/form_fields/TextField.vue

```diff
@@ -56,4 +58,8 @@ const onUpdateModel = ( newValue: string|number ): void => {
 
 <style scoped lang="scss">
 
+.field-info-message {
+	color: rgba(0,0,0,.6);
```

Comment:
Please use an actual grey color here rather than a low opaque black. You can import the colors with:

```scss
@use '@src/scss/settings/colors';
```